jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/SignalStatusListBuilder.java

```diff
@@ -14,21 +14,21 @@ private SignalStatusListBuilder() {
 	public static J2735SignalStatusList genericSignalStatusList(JsonNode status) {
 		J2735SignalStatusList signalStatusList = new J2735SignalStatusList();
 
-		if (status.isArray()) {
-			Iterator<JsonNode> elements = status.elements();
-
-			while (elements.hasNext()) {
-				signalStatusList.getStatus()
-                    .add(SignalStatusBuilder.genericSignalStatus(elements.next()));
-			}
-		} else {
-			JsonNode signalStatus = status.get("SignalStatus");
-			if(signalStatus != null)
-			{
-				signalStatusList.getStatus()
-					.add(SignalStatusBuilder.genericSignalStatus(signalStatus));
-			}
-		}
+        JsonNode signalStatus = status.get("SignalStatus");
+        if(signalStatus != null)
+		{
```

Comment:
polish(non-blocking): consistent if block style
```suggestion
        if(signalStatus != null) {
```

Reply:
They should now be consistent now.